contents/uix/design-systems.md

```diff
@@ -0,0 +1,130 @@
+
+**Storybook**
+
+<center>
+<img src="https://raw.githubusercontent.com/storybooks/storybook/next/media/storybook-intro.gif" alt="Storybook" width="95%">
+
+_Figure 4. Storybook_ ([source](https://github.com/storybooks/storybook))
+</center>
+
+[Storybook](https://github.com/storybooks/storybook) is a development environment for UI components. It helps you build UI components in isolation and offers handy features to visualize any mock data you supply. It helps in setting up the pattern library of a Design System.
+
+**React Styleguidist**
+
+<center>
+<img src="https://camo.githubusercontent.com/0ded3b0835a4ace4664a6833985affbde783ed47/68747470733a2f2f64337676366c703535716a6171632e636c6f756466726f6e742e6e65742f6974656d732f303631663041326e3142304833703054317031662f72656163742d7374796c65677569646973742d6c6f676f2e706e67" alt="React Styleguidist" width="60%">
+
+_Figure 5. React Styleguidist_ ([source](https://github.com/styleguidist/react-styleguidist))
+</center>
+
+[React Styleguidist](https://github.com/styleguidist/react-styleguidist) simplifies creating and maintaining a UI documentation site. It allows you to create pages in Markdown and import UI components. It helps in setting up the style guide of a Design System.
+
+## Examples of existing Design Systems
+
+This [website](https://designsystemsrepo.com/design-systems/) presents a comprehensive and curated list of design systems.
+
+Out of the list, there are some noteworthy open-source Design Systems that you can use, contribute or take reference from:
+1. [Material Design](https://material.io/design/): Created by Google. Components implemented from Material Design are found [here](https://github.com/material-components).
+2. [Photon Design System](https://design.firefox.com/photon/): Used to build Firefox products. Components found [here](https://github.com/firefoxux).
+3. [Lightning Design System](https://www.lightningdesignsystem.com/): Used to build products/apps related to Salesforce. The component library can be found [here](https://github.com/salesforce/design-system-react). Note that they use Storybook in setting up the pattern library.
+4. [Shopify Polaris](https://polaris.shopify.com/): Used to build products/apps related to Shopify. The component library can be found [here](https://github.com/Shopify/polaris-react). Note that they use Storybook in setting up the pattern library.
+
+## Ending Notes
+
+Design Systems keep things in order by acting as a Single Source of Truth, providing a common shared language across the organization. It encourages reusability of components, resulting in faster development time. Consider building a Design System in your next company or contribute to one now.
```

Comment:
We don't usually have this kind of a call to action in other articles. Omit for the sake of consistency?